swap/src/protocol/bob/swap.rs

```diff
@@ -29,6 +29,10 @@ pub fn is_btc_locked(state: &BobState) -> bool {
     matches!(state, BobState::BtcLocked(..))
 }
 
+pub fn is_lock_proof_received(state: &BobState) -> bool {
```

Comment:
This is only used in tests, right? Please don't add test code to release code :)

Reply:
I moved all of those `is_xxx_state` funtcions into testutils